tensorbay/client/dataset.py

```diff
@@ -699,10 +699,10 @@ class FusionDatasetClient(DatasetClientBase):
 
     @staticmethod
     def _skip_upload_frame(
-        segment_filter: Iterator[Tuple[Frame, Optional[int], bool]],
+        segment_filter: Iterator[Tuple[Frame, Optional[float], bool]],
```

Comment:
better to paste the mypy error detail in commit message body

Reply:
- Put the error detail into code block in commit message body
- `error` -> `errors` in commit message